### DISCUSSION TITLE
Mattermost Plugin

### DISCUSSION THREAD

**Discussion by aurindam** [girish17/op-mattermost#60] [Ideas]
Any plans to make this plugin available on Mattermost marketplace? https://mattermost.com/marketplace/

**Comment by girish17** [comment]
Hi @aurindam ,

This project is an integration for OpenProject and Mattermost. The OpenProject plugin project is available [here](https://github.com/girish17/op-mm-plugin) .
The submission for the same has been made to the Mattermost marketplace. The team from Mattermost needs to review it before it becomes available there. Until then, you can download it [here](https://github.com/girish17/op-mm-plugin/releases/tag/v0.2) and use it.

Thanks,
girish
